capability.go

```diff
@@ -879,13 +881,21 @@ type Returner interface {
 	// ReleaseResults is called.
 	AllocResults(sz ObjectSize) (Struct, error)
 
-	// Return resolves the method call successfully if e is nil, or failure
-	// otherwise.  Return must be called once.
+	// PrepareReturn finalizes the return message. The method call will
+	// resolve successfully if e is nil, or otherwise it will fail.
```

Comment:
Nit:  I initially misunderstood this to mean "the actual sending of the return message will fail".  Consider updating docstring to something like:

>The method call will resolve successfully if e is nil.  Otherwise, it will return an exception to the caller.